guides/common/modules/con_permissions-for-remote-execution.adoc

```diff
@@ -15,6 +15,13 @@ You can use the `view_hosts` and `view_smart_proxies` permissions to limit which
 The `execute_template_invocation` permission is a special permission that is checked immediately before execution of a job begins.
 This permission defines which job template you can run on a particular host.
 This allows for even more granularity when specifying permissions.
+
+ifdef::satellite[]
```

Comment:
@adamruzicka is this not possible on Foreman/Katello?

Reply:
It is possible. There is no difference in this area between foreman/katello and satellite